flink-table/flink-sql-parser/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -1348,6 +1352,29 @@ SqlCreate SqlCreateTable(Span s, boolean replace, boolean isTemporary) :
         String p = SqlParserUtil.parseString(token.image);
         comment = SqlLiteral.createCharString(p, getPos());
     }]
+    [
+        <DISTRIBUTED>
+        (
+            <INTO> { bucketCount = Literal(); } <BUCKETS>
```

Comment:
Should we rather use `UnsignedNumericLiteral`, so that we ensure it's numeric on the parser level.

Reply:
Will do!